engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -466,9 +467,14 @@ case class WorkflowActor(workflow: WorkflowDescriptor)
       val updatedData = startRunnableCalls(data)
       self ! CheckForWorkflowComplete
       stay() using updatedData
-    case Event(CallStarted(callKey), data) if !data.isPending(callKey) =>
+    case Event(CallStarted(callKey, maybeCallLogs), data) if !data.isPending(callKey) =>
       executionStore += callKey -> ExecutionStatus.Running
-      persistStatus(callKey, ExecutionStatus.Running)
+      // TODO: We're not using/waiting for the future results
+      // TODO: This should be in a single transaction!
```

Comment:
Heh, nice. Let's discuss post standup where it may be easier to sketch out.

I do see additional `cromwell.engine.db.slick` methods returning `DBIO[_]` objects, such that they could be more easily composed into different transactions. But I feel that anywhere we have a composed database `Future` should be encapsulated and pushed down into `cromwell.engine.db` methods, instead of pulled up into the actors, as part of SoC.